geode-core/src/main/java/org/apache/geode/management/internal/web/controllers/ShellCommandsController.java

```diff
@@ -53,7 +53,18 @@
 /**
  * The ShellCommandsController class implements GemFire REST API calls for Gfsh Shell Commands.
  * 
- * @see org.apache.geode.management.internal.cli.commands.ShellCommands
+ * @see org.apache.geode.management.internal.cli.commands.ConnectCommand
+ * @see org.apache.geode.management.internal.cli.commands.DebugCommand
```

Comment:
I think we should get rid of all those @see. They are not even related to this controller anymore.